fieldservice_sale/views/product_template.xml

```diff
@@ -5,7 +5,7 @@
         <field name="inherit_id" ref="sale.product_template_form_view_invoice_policy" />
         <field name="arch" type="xml">
             <xpath expr="//field[@name='service_type']" position="after">
-                <field name="field_service_tracking" widget="radio" />
+                <field name="field_service_tracking" widget="radio" attrs="{'invisible': [('type', '!=', 'service')]}" />
```

Comment:
@hparfr In my use case we need to create field service orders for storable type products as well as service type products.  

```suggestion
                <field name="field_service_tracking" widget="radio" />
```

Reply:
@brian10048 What is your use case ?

How is it supposed to work regarding the "shipped qty" invoice policy ? What about returned products / Partial delivery (backorders) / Qty != 1 / Legal stuff about customs...

Reply:
@hparfr

We have a product such as a tree or plant.

The only way we sell the tree to a customer is by delivering the tree to the location and providing a service to install the tree at the customer location. We still need to track the stock moves of the tree from our vendor, to the worker's truck, and to the customer.  We also need the FSM order to track and provide the service of installation.

Shipped qty policy should still be working.  If the stock moves are confirmed on the linked field service order, the sale order delivery quantity is updated.

I don't really have many situations for returned products or partial delivery etc. so I do not address these issues you may have.

I'm curious how you would configure this for my scenario.  I saw another comment where you mention using product packs?

Reply:
>I'm curious how you would configure this for my scenario. I saw another comment where you mention using product packs?

Yes https://github.com/OCA/product-pack or  https://github.com/akretion/sale-configurator may achieve what you need.
They will ensure you have on the sale order : the tree (stockable product) and installation (service) and they can ensure the quantities are increased for both lines (x trees -> x installations).
![Screenshot_2021-02-25 Tree installation - Odoo](https://user-images.githubusercontent.com/8331206/109196967-b0d0a780-779c-11eb-8b1d-8a4db827eaf7.png)
![Screenshot_2021-02-25 S00029 - Odoo(1)](https://user-images.githubusercontent.com/8331206/109196979-b6c68880-779c-11eb-9611-6abbd79d4791.png)

Reply:
So what is your position with fielrd_service_tracking ?

From my point of view, by default it should be invisible when not a service.
And your need can be resolved thanks to product_packs , sale_configuratior or simply a custom module changing the view.

But if it really matters for you, to keep it always visible, I will accept your commit suggestion :dagger:  :-)

Reply:
We can move forward with this as it is and make invisible when not a service